app/prj.conf

```diff
@@ -122,10 +122,13 @@ CONFIG_SMF_INITIAL_TRANSITION=y
 # Location
 CONFIG_LOCATION=y
 CONFIG_LOCATION_DATA_DETAILS=y
-CONFIG_LOCATION_METHOD_GNSS=n
+CONFIG_LOCATION_METHOD_GNSS=y
+CONFIG_LOCATION_SERVICE_NRF_CLOUD_GNSS_POS_SEND=y
+CONFIG_LOCATION_SERVICE_HERE=n
 CONFIG_LOCATION_REQUEST_DEFAULT_GNSS_TIMEOUT=200000
 CONFIG_LOCATION_REQUEST_DEFAULT_TIMEOUT=300000
 CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS=y
+# CONFIG_LOCATION_LOG_LEVEL_DBG=y
```

Comment:
Unused, should we just remove it?

Reply:
I've been turning it on and off multiple times, and I figured it would be handy to have there easily available during development. We can remove it before launch.